### DISCUSSION TITLE
zh/docs/deployment/deploy-cdn

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#1730] [Docs comments]
# zh/docs/deployment/deploy-cdn

在CDN中托管前端静态文件

https://casdoor.org/zh/docs/deployment/deploy-cdn

**Comment by yasinshaw** [comment]
这里如果用docker是不是不太好配置。。还要去容器里面改东西。

**Reply by gongyan200** [reply]
有个简单的方法，在caddy或者nginx反向代理中，将所有的/static*请求，301重定向到CDN。
以caddy配置为例，只需将js、css文件放到CDN上后，再加如下一条redir规则
casdoor.example.com {
    redir /static* https://cdn.example.com{uri} 301
    reverse_proxy :8000
}

**Reply by gongyan200** [reply]
配置显示错了
```
casdoor.example.com {
redir /static* https://cdn.example.com{uri} 301
reverse_proxy :8000
}
```
